docs/github.md

```diff
@@ -65,7 +65,8 @@ able to create a branch for a revision.
 $ # Do your work
 $ jj commit
 $ # Jujutsu automatically creates a branch
-$ jj git push --change $revision
+$ jj git push --change $revision # Here $revision refers to any of these variants
+$ # the full commit id or change id or the highlighted commit or change id.
```

Comment:
nit: Either indent subsequent lines to align with `# Here...`, or move it to a comment after the line (probably simpler). Also missing a colon after "variants". Actually, should it just point to the revset documentation instead? Or just say something like this:

> "$revision" can be a change id, for example

Reply:
See the comment [here], maybe you can suggest something better. 

[here]: https://github.com/martinvonz/jj/issues/1989#issuecomment-1692241629

Reply:
How about `<revision>` then? `<X>` seems like a pretty ubiquitous way of saying "insert your value for X here". Sounds good, @joyously?

Reply:
No, actually. If you are giving example commands, don't put variables in there. Put the actual command as it would be typed (or copy/pasted).
Aside: the `<` character is problematic from both the shell and the HTML point of view.

Reply:
How about something like this then?
```
# Push change "mw", letting Jujutsu automatically create a branch called "push-mwmpwkwknuzk"
$ jj git push --change mw
```

Reply:
That sounds good.

Reply:
Done now.